content/SCALE/SCALECLIReference/Account/CLIgroup.md

```diff
@@ -10,8 +10,221 @@ tags:
 
 {{< toc >}}
 
-
 {{< include file="/_includes/CLI/CLIGuideWIP.md" type="page" >}}
 
+The **group** namespace contains 8 commands and is based on functions found in the SCALE API and web UI. It provides access to group account creation, configuration, and management.
+
+You can enter commands from the main CLI prompt or from the system namespace prompt.
+
+## Create Command
+The `create` command creates a new group.
+
+{{< include file="content/_includes/CLI/AccountGroupCreateExample.md" type="page" >}}
+
+### Create Interactive Arguments Editor
+
+The `create --` option opens an **interactive arguments editor**.
+
+{{< expand "Using the create Interactive Arguments Editor" "v" >}}
+{{< trueimage src="/images/SCALE/CLI/Account/GroupCreateInteractiveArgumentsEditor.png" alt="Interactive Arguments Editor" id="1: Interactive Arguments Editor" >}}
+
+The interactive arguments editor provides a text user interface (TUI) where you can configure arguments and options. The TUI also provides some information on required arguments, defaults, and expected input types (string, boolean, integer, or array).
+
+In the TUI, most arguments are initially marked as comments with the `#` symbol, indicating that they are not yet configured. However, `name:` is shown as a required field.
+
+To provide values for the other arguments, you need to remove the `#` comment designator from the corresponding line in the TUI.
+
+{{< hint type=tip >}}
+A group name is the only required value for group creation. If a group identification number is not provided, it is automatically filled with the next one available. All other arguments are optional.
+
+See the relevant rows in [Configuration Arguments](#create-configuration-arguments) for more information.
+{{< /hint >}}
+
+A space is required between the provided argument and entered data, for example `name: test_group`.
+
+Press <kbd>F2</kbd> or click **Save** to save the modified file.
+
+Press <kbd>F10</kbd>, <kbd>Esc</kbd>, or click **Quit** to exit the TUI.
+The `create` command automatically executes upon exit.
+{{< /expand >}}
+
+### Create Configuration Arguments
+
+`create` has 7 available arguments for group configuration. They are:
```

Comment:
```suggestion
`create` has seven available arguments for group configuration. They are:
```
I see the table below with the arguments and description details. Maybe remove this bullet list and tell them to see the table below.

Reply:
My thought was this gives simplified version so that the list is not completely buried in the expand, but then there is also more detail there